src/odbc/odbc.c

```diff
@@ -4785,11 +4785,27 @@ ODBC_FUNC(SQLColumns, (P(SQLHSTMT,hstmt), PCHARIN(CatalogName,SQLSMALLINT),	/* o
 	WIDE))
 {
 	int retcode;
+	DSTR catalog_name = DSTR_INITIALIZER;
+	SQLSMALLINT catalog_size;
+	const char *proc = NULL;
```

Comment:
Why not initializing to `"sp_columns"` directly here?

Reply:
Good feedback; will change, thanks.

Reply:
Done, let me know if I missed anything.